### ISSUE TITLE
Find out how to intialize the weights the best

### ISSUE THREAD

**Issue by dbeinhauer** [dbeinhauer/mcs-source#54]
While using the model we sometimes encounter the problems with the training, gradient overflow and so on. We partially solved these problems using gradient clipping but there might be much better improvement of the model if we initialize the weights in better way. This approach might significantly improve the model training.

**Comment by dbeinhauer** [comment]
It seems that using default Pytorch initialization is enough. We did not encounter improvements in the custom weights initialization.
